### PR TITLE
docs(buttons): use variant prop for buttons in live code snippets

### PR DESCRIPTION
**Pull request by adamstankiewicz**

### CODE REVIEW

File: www/src/pages/components/modal.mdx [hunk 1]
```diff
@@ -63,12 +63,12 @@ class ModalWrapper extends React.Component {
           parentSelector={this.props.parentSelector}
           onClose={this.resetModalWrapperState}
           buttons={[
-            <Button buttonType="success">Green button!</Button>
+            <Button variant="success">Green button!</Button>
```
Comment (abutterworth) [inline www/src/pages/components/modal.mdx:66]: Modal still uses the old button I think

Comment (abutterworth) [reply www/src/pages/components/modal.mdx:66]: I take it back :P it looks like it's using the new one.

**Review by abutterworth — APPROVED** [review]
I think if you

**Review by muselesscreator — COMMENTED** [review]
Nice concise change.  LGTM

**Pull request merged**